src/test/functional/features/check-your-answers.feature

```diff
@@ -41,7 +41,8 @@ Feature: Check Your Answers
     And the page should include "Payment"
     And the page should include "Do you need help paying the fee for your divorce?	I need help paying the fee"
     And the page should include "Have you already applied for help with your divorce fee?	Yes"
-    And the page should include "HWF-ABC-123"
+    # @TODO
```

Comment:
how comes this is commented out? might be more helpful for someone who picks it up later to have a comment on it.

Reply:
same to the other ones.

Reply:
👍 This has been fixed in another PR which has been recently merged, so I've reverted these changes in 32740b2.